src/emulator/commandUtils.ts

```diff
@@ -276,17 +276,19 @@ function processKillSignal(
         const pids: number[] = [];
 
         const emulatorsTable = new Table({
-          head: ["Emulator", "Host:Port", "PID"],
+          head: ["Emulator", "Host", "Port", "PID"],
```

Comment:
Similar here - this will make things un-clickable. Could we maintain that?

Reply:
Good catch! I've restored the behavior. Added some comments too for background.